db/migrate/20230328131527_add_users_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :users, null: false, foreign_key: true
```

Comment:
- [ ] Similarly, considering the given ERD, `author` is aliasing the `user` model, and hence, our migration query should be similar to:

```ruby
...
  def change
    add_reference :comments, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...

```

or.. 

```ruby
  create_table :comments do |t|
    . . .
    t.references :author, foreign_key: { to_table: 'users' } 
    . . .
  end
```

Reply:
Hii @rloterh 
According to the requirements and the given ERD, we only have to name author_id in the posts table, not in other tables.

Reply:
Hi @gauravgangwarweb

Reply:
**Hi, fellow Code Reviewer,**

This project is complete, kindly approve it!

Thanks in advance!😃